src/controllers/v1/attachment/index.ts

```diff
@@ -71,6 +41,36 @@ export class attachment extends Controller {
     this.log = logger.child({ controller: '/attachment' })
   }
 
+  parseAccept = (acceptHeader: string) =>
+    acceptHeader
+      .split(',')
+      .map((acceptElement) => {
+        const trimmed = acceptElement.trim()
+        const [mimeType, quality = '1'] = trimmed.split(';q=')
+        return { mimeType, quality: parseFloat(quality) }
+      })
+      .sort((a, b) => {
+        if (a.quality !== b.quality) {
+          return b.quality - a.quality
+        }
+        const [aType, aSubtype] = a.mimeType.split('/')
+        const [bType, bSubtype] = b.mimeType.split('/')
+        if (aType === '*' && bType !== '*') {
+          return 1
+        }
+        if (aType !== '*' && bType === '*') {
+          return -1
+        }
+        if (aSubtype === '*' && bSubtype !== '*') {
+          return 1
+        }
+        if (aSubtype !== '*' && bSubtype === '*') {
+          return -1
+        }
+        return 0
+      })
+      .map(({ mimeType }) => mimeType)
```

Comment:
if we're moving this should we move this to a util? I actually think this is a great candidate for an npm lib at some point

Reply:
Indeed it's, moved to a utils file